src/balena.sbvr

```diff
@@ -998,3 +1001,5 @@ Rule: It is necessary that each release that should operate a device that is of
 Rule: It is necessary that each release that should manage a device, has a status that is equal to "success" and has a semver major that is greater than 0 or has a semver minor that is greater than 0 or has a semver patch that is greater than 0.
 -- The first part of this rule is meant to prevent accidentally setting a host extension as a supervisor (i.e., another public + non-host app)
 Rule: It is necessary that each release that should manage a device that is of a device type1, belongs to an application that is public and is not host and has a slug that is equal to "balena_os/aarch64-supervisor" or "balena_os/amd64-supervisor" or "balena_os/armv7hf-supervisor" or "balena_os/i386-supervisor" or "balena_os/i386-nlp-supervisor" or "balena_os/rpi-supervisor" and is for a device type2 that is of a cpu architecture that is supported by the device type1.
+Rule: It is necessary that each application that is not host, is updated by no application.
+Rule: It is necessary that each application that is of a class that is not equal to "block", updates no application.
```

Comment:
I'm worried about the performance in general as I've seen in the past that self-referential joins can be very painful on pg performance